src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reversed = {};
+  const values = [];
+
+  for (const key in robot) {
+    if (values.indexOf(robot[key]) === -1) {
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 2nd item, there you can see how to properly check if the property is present in object